packages/babel-plugin-transform-modules-systemjs/src/index.js

```diff
@@ -547,6 +545,17 @@ export default declare((api, options) => {
             path.remove();
           }
 
+          let hasTLA = false;
+          path.traverse({
+            AwaitExpression(path) {
+              hasTLA = true;
+              path.stop();
+            },
+            Function(path) {
+              path.skip();
+            },
```

Comment:
```suggestion
            },
            noScope: true,
```
nit: Use `noScope: true` should be faster since `path.scope` is not used in visitors.